### DISCUSSION TITLE
I want to make a gallery app

### DISCUSSION THREAD

**Discussion by cjwolf49** [yasirkula/UnityNativeGallery#273] [Q&A]
I want to make a gallery app, as you can see in the picture while uploading the Instagram feed.
In the case of Android, the file in the DCIM path was created using the NativeGallery.LoadImageAtPath using System.IO.
How can I make an iPhone?
Is there a function that can get the list in the current device's photo library?

**Comment by yasirkula** [comment]
There are native Objective-C functions that can be used to get the image list but it will require full Photos permission granted and the returned image list won't consist of file paths. You'll need to copy those images to temporaryCachePath before being able to create Texture2Ds from them. As for what those functions are, I don't know :) You can google "ios objective-c get all images" to learn more. Just be aware that some code snippets online are deprecated.
